core/src/main/java/nu/yona/server/subscriptions/service/UserDTO.java

```diff
@@ -154,17 +154,12 @@ static UserDTO createInstance(User userEntity)
 	{
 		if (userEntity == null)
 		{
-			return createRemovedUserInstance();
+			return null;
```

Comment:
Can't you fix this at the level of the caller? It's a weird contract to pass a null value to a ``createInstance`` method and then get null back.